cpp/src/parquet/encoding_test.cc

```diff
@@ -862,5 +862,208 @@ TEST_F(DictEncoding, CheckDecodeIndicesNoNulls) {
   CheckDict(actual_num_values, *builder);
 }
 
+// ----------------------------------------------------------------------
+// BYTE_STREAM_SPLIT encode/decode tests.
+
+namespace {
```

Comment:
An anonymous namespace isn't really useful in a test executable.

Reply:
Agreed

Reply:
Removed.